Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ErrorMessageFactory.swift

```diff
@@ -29,6 +29,11 @@ internal enum ErrorMessageFactory {
                 View is too large to render monochrome snapshot. Try setting useMonochromeSnapshot to false or use a \
                 different iOS version. In particular, this is known to fail on iOS 13, but was fixed in iOS 14.
                 """
+        case AccessibilitySnapshotView.Error.containedViewHasUnsupportedTransform:
```

Comment:
Same comment as the other PR...we should move these error message strings into a centralized spot. Non-blocking for sure.

Reply:
Same issue here with the current module boundaries making that tricky. It's split up here the same way (shared for the Swift and Objective-C implementations of iOSSnapshotTestCase, but separate for the SnapshotTesting implementation), but I'll think about whether there's a way to combine those.